arm-network/2016-06-01/swagger/network.json

```diff
@@ -4288,6 +4288,27 @@
       },
       "description": "SKU of application gateway"
     },
+    "ApplicationGatewaySslPolicy": {
+      "properties": {
+        "disabledSslProtocols": {
+          "type": "array",
+          "description": "Get or sets SSL protocols to be disabled on Application Gateway",
```

Comment:
Please remove Get or Set from the description of all the model properties. Autorest will add it automatically as needed.